pkg/model/keycloak_ingress.go

```diff
@@ -23,6 +23,7 @@ func KeycloakIngress(cr *kc.Keycloak) *networkingv1.Ingress {
 				"app": ApplicationName,
 			},
 			Annotations: map[string]string{
+				"kubernetes.io/ingress.class":                  "nginx",
```

Comment:
we are forcing the class of the ingress...... would this cause problems if the customer doesnt have nginx and has another controller installed ?
I think we should not enforce any specific class for ingress, storage, etc.